pipeline/remove_subscribers.py

```diff
@@ -66,6 +69,20 @@ def main_remove_subscriptions() -> None:
     logging.info("Connection to database successfully closed.")
 
 
+def lambda_handler(event, context):
+    """Lambda handler function """
+    load_dotenv()
+    logging.info(f"Attempting to remove subscriptions at: {datetime.now()}.")
```

Comment:
Good idea to log the exact time subscribers were removed, will be useful info if something goes wrong